src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Review from './review.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div>
+        {/*<Review />*/}
+        hello everybody
+        <Review />
```

Comment:
I added the id of the item as the key for review so that it would would be re-rendered when its state was updated